### PR TITLE
Column visibility dropdown list

### PR DESCRIPTION
**Pull request by HansPinckaers**
*can be tested now*

There are some assumptions in this code:
- [x] the most important metric always is the first metric-column
- [x] Datatable crashes if there no column to be toggled.

### CODE REVIEW

**Comment by jmsmkn** [comment]
I assume that we will only want the extra results columns to be toggleable, we know how many there are from `{{ extra_results_columns|length }}`, and they always start at column 4 (zero index).

**Comment by HansPinckaers** [comment]
I was thinking only to allow the metrics to be toggable and always show the supplemental information / comments?

**Comment by jmsmkn** [comment]
I agree - only for extra_results_columns though. The main result should always be visible.

**Comment by HansPinckaers** [comment]
It looks quite clean now. Should be tested in staging environment.

toggable

File: app/grandchallenge/evaluation/templates/evaluation/result_list.html [hunk 1]
```diff
@@ -199,11 +199,50 @@ <h2>Results</h2>
...
+                buttons: [
+                    {
+                        extend: 'colvis',
+                        text: 'Additional metrics',
+                        columns: '.toggable'
+                    },
+                    {
+                        text: 'Show all metrics',
+                        action: function ( e, dt, node, config ) {
+                            if($(node).text().indexOf('Hide') > -1) {
```
Comment (HansPinckaers) [inline app/grandchallenge/evaluation/templates/evaluation/result_list.html]: I do not really like using HTML of the button to determine 'state', maybe use an attribute of a certain element.

**Pull request merged**